polkadot-protocol/architecture/system-chains/people.md

```diff
@@ -0,0 +1,103 @@
+---
+title: People
+description: Learn how People Chain secures decentralized identity management, empowering users to control and verify digital identities without central authorities.
+---
+
+# People
+
+## Introduction
+
+People Chain is a specialized parachain within the Polkadot ecosystem dedicated to secure, decentralized identity management. 
+
+This solution empowers users to create, control, and verify their digital identities without reliance on centralized authorities. By prioritizing user sovereignty and data privacy, People Chain establishes a foundation for trusted interactions throughout the Polkadot ecosystem while returning control of personal information to individuals.
+
+## Identity Management System
+
+People Chain provides a comprehensive identity framework allowing users to:
+
+- Establish verifiable on-chain identities
+- Control disclosure of personal information
+- Receive verification from trusted registrars
+- Link multiple accounts under a unified identity
+
+Users must reserve funds in a bond to store their information on chain. These funds are locked, not spent, and returned when the identity is cleared.
+
+### Sub-Identities
+
+The platform supports hierarchical identity structures through sub-accounts:
+
+- Primary accounts can establish up to 100 linked sub-accounts
+- Each sub-account maintains its own distinct identity
+- All sub-accounts require a separate bond deposit
+
+## Verification Process
+
+### Judgment Requests
+
+After establishing an on-chain identity, users can request verification from registrars:
```

Comment:
First time mentioning a registrar, so we should briefly intro what a registrar is